src/assets/js/protocol/protocol-details.js

```diff
@@ -0,0 +1,229 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+// create namespace
+if (typeof Mozilla === 'undefined') {
+    var Mozilla = {};
+}
+
+(function(doc) {
+    'use strict';
+
+    var Details = {};
+    var _count = 0;
+
+    /**
+     * isSupported
+     * - feature detection for HTML5 detail/summary support
+     * - // https://mathiasbynens.be/notes/html5-details-jquery#comment-35
+     */
+    Details.isSupported = (function() {
+        var el = doc.createElement('details');
+        var fake;
+        var root;
+        var diff;
+        if (!('open' in el)) {
```

Comment:
Nit: could add some spacing to this code just to make it a little more readable?